docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -727,6 +727,23 @@ Your new custom cloud is ready to use only after its status changes to
      --display-name "CUSTOM_CLOUD_DISPLAY_NAME"
    ```
 
+   where
+
+   - `organization-id` is the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud.
+   - `deployment-model` is the type of [network architecture](/docs/platform/concepts/byoc#byoc-deployment)
+     your custom cloud uses:
+     - `public` model allows the Aiven control plane to connect to the service nodes via the
+       public Internet.
+     - `private` model routes traffic through a proxy for additional security utilizing a
+       bastion host physically separated from the Aiven services.
+   - `cloud-provider` is the name of a cloud provider to be used, which is `google`.
+   - `cloud-region` is the name of a Google region where to create your custom cloud.
+   - `reserved-cidr` is a CIDR block defining the IP address range of the VPC that Aiven
+     creates in your own cloud account, for example: `10.0.0.0/16`, `172.31.0.0/16`, or
+     `192.168.0.0/20`.
+   - `display-name` is the name of your custom cloud.
```

Comment:
It's a bit confusing that the variable here has a different name and case than in the script above.
`CUSTOM_CLOUD_DISPLAY_NAME` vs `display-name`